packages/moment/src/moment-utils.ts

```diff
@@ -2,6 +2,7 @@ import defaultMoment from "moment";
 import { IUtils, DateIOFormats } from "@date-io/core/IUtils";
 
 interface Opts {
+  useLocalizedFormats?: boolean;
```

Comment:
Do we need a flag? What about making it the default?

Reply:
It is only for test